### ISSUE TITLE
Option to enable full lines at the end of that page

### ISSUE THREAD

**Issue by MrAndre96** [superolmo/BigFiles#20]
Currently with the pages it will stop when it reached a certain amount of characters.
I would like a option to make it that it takes everything of the last line instead of stopping in the middle of that line.

**Comment by superolmo** [comment]
Yes, that is something that I was thinking of adding, but it is not as easy. Some files uses Carriage Return and Line Feed, and others use Line Feed only. Also, I need to recalculate the total number of pages as it will change when I retrieve less than buffer size. I am adding it to the next release for now.
